app/ui/src/app/platform/types/integration/integration.models.ts

```diff
@@ -58,7 +68,7 @@ export interface IntegrationDeployment extends BaseEntity {
   lastUpdated: number;
   integrationId: string;
   version: number;
-  currentState: IntegrationStatus;
+  currentStatus: IntegrationStatus;
```

Comment:
Here too, and feel free to ignore this if this is something that's going to change in the backend too...

Reply:
Oh damn! good point. I originally changed the property because I wanted it to match the type name (`IntegrationState` could have to be renamed - otherwise it clashed against the `xxxState` token), but forgot that this is likely coming from the backend. Will reinstate the original values again

Reply:
Done!